app/views/settings/preferences/show.html.haml

```diff
@@ -44,6 +47,3 @@
 
   .fields-group
     = f.input :setting_auto_play_gif, as: :boolean, wrapper: :with_label
-
-  .actions
-    = f.button :button, t('generic.save_changes'), type: :submit
```

Comment:
makes sense

I don't think it hurts to have it in both places

@foxsan48 please add it back at the bottom~